src/app/debug/table/table-settings-modal/table-settings-modal.component.html

```diff
@@ -38,7 +38,7 @@ <h5>Editor</h5>
         <h5>Report table</h5>
         <div title="Configure the amount of spacing in the report table">
           <label for="show-multiple">Spacing for report table</label>
-          <select #spacingOptionsDropdown data-cy-spacing-dropdown-input class="form-control custom-select mr-sm-2"
+          <select #spacingOptionsDropdown data-cy-settings-modal="spacingDropdown" class="form-control custom-select mr-sm-2"
```

Comment:
Modal mag je hier weglaten
```suggestion
          <select #spacingOptionsDropdown data-cy-settings="spacingDropdown" class="form-control custom-select mr-sm-2"
```